src/iconv.cr

```
@@ -8,7 +8,7 @@ struct Iconv
     original_from, original_to = from, to
 
     @skip_invalid = invalid == :skip
-    {% unless flag?(:freebsd) %}
+    {% unless flag?(:freebsd) || flag?(:musl) %}
```

This is out of the scope of the PR but is failing silently here a good idea?